src/index.js

```diff
@@ -69,7 +71,27 @@ if (program.sam) {
   provider = "SAM";
 } else {
   provider = "serverlessFramework";
+  const SLS = new Serverless();
+  if (!program.stackName) {
+    program.stackName = SLS.getStackName(program.stage);
+  }
+
+  if (!program.region) {
+    program.region = SLS.getRegion();
+  }
 }
+
+if (!program.stackName) {
+  console.error(
+    "error: required option '-n, --stack-name <stackName>' not specified"
+  );
+  process.exit(1);
+}
+if (!program.region) {
+  console.error("error: required option '-r, --region <region>' not specified");
+  process.exit(1);
```

Comment:
@DotGav how does this link to the region navigation feature you're working on - should we exit when no region passed?

Reply:
Makes sense to have a defined error until we have a startup wizard to select the region and the stack